From d52e9363172ce3b260b66a9d8639af6e6a0c298d Mon Sep 17 00:00:00 2001
From: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Date: Tue, 12 Mar 2019 17:57:26 +0200
Subject: [PATCH 464/741] fsl_qbman: add dummy functions for probe deferal APIs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit 43d04338e265cea4f287853576fdc7a872fefc40 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add missing definitions for probe deferal APIs in order
to fix these compilation issues for drivers that use them
with the SDK version of the qbman drivers:

drivers/crypto/caam/ctrl.c: In function ‘caam_probe’:
drivers/crypto/caam/ctrl.c:544:9: error: implicit declaration of function ‘qman_is_probed’
drivers/crypto/caam/ctrl.c:554:9: error: implicit declaration of function ‘qman_portals_probed’

Signed-off-by: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/linux/fsl_qman.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/include/linux/fsl_qman.h b/include/linux/fsl_qman.h
index 56c05fdb9927..e39a17c2b6e5 100644
--- a/include/linux/fsl_qman.h
+++ b/include/linux/fsl_qman.h
@@ -3893,6 +3893,16 @@ int qman_p_enqueue_orp(struct qman_portal *p, struct qman_fq *fq,
 int qman_p_enqueue_precommit(struct qman_portal *p, struct qman_fq *fq,
 				const struct qm_fd *fd, u32 flags,
 				qman_cb_precommit cb, void *cb_arg);
+
+static inline int qman_is_probed(void) {
+	return 1;
+}
+
+
+static inline int qman_portals_probed(void) {
+	return 1;
+}
+
 #ifdef __cplusplus
 }
 #endif
-- 
2.17.1

